phys/module_sf_bem.F

```diff
@@ -1128,7 +1383,7 @@ subroutine int_rlrad(emwal,emwin,sigma,twal_int,twin,&
   			  
 	call gaussjbem(a,6,b,6)
 
-        do iw=1,6
+            do iw=1,6
```

Comment:
white space - previous code better lined up

Reply:
JImy,
Agreed, I have lined the "do" up with the "end do"